src/main/reactjs/src/interfaces/authorisation.ts

```diff
@@ -12,6 +12,7 @@ export interface AuthorisationTerm extends WithId, WithVersion {
 export interface Authorisation extends WithId, WithVersion {
```

Comment:
Tässä Autorization voisi extendata APIAuthorisationin?

Reply:
Saako tuon miten toteutettua? Sama `Omit<>` ratkaisu kun muualla ei vaikuttanut toimivan jos useampi kenttä korvataan toisella.

Reply:
Sen saa toteutettua monella ei tavalla, mutta varmaa omit ratkaisu olisi näin -->

```
export interface Authorisation
  extends Omit<
    APIAuthorisation,
    'autDate' | 'virDate' | 'assuranceDate' | 'meetingDate' | 'terms'
  > {
  autDate?: Date;
  virDate?: Date;
  assuranceDate?: Date;
  meetingDate?: Date;
  effectiveTerm?: AuthorisationTerm;
  terms?: Array<AuthorisationTerm>;
}
```

En tie onko tämä yhtään parempi, mutta toisaalta tästä tietää ainakin että ne liittyvät toisiinsa ja jos tehdään muutoksia interfaceen APIAuthorisation typescript herjaa niistä.

Reply:
Ihan pätevä minusta

Reply:
Joo, voisihan tuo olla käytössä sielläkin. Voi sitten miettiä samalla että pitäisikö backendin palauttaa data mieluummin eri formaatissa / tulisiko sieltä `start` ja `end` nykyisellä tavalla nimettyjen muuttujien sijaan.